.github/workflows/workflow.yml

```diff
@@ -0,0 +1,78 @@
+# Builds and performs various check
+
+---
+name: Build
+
+on: 
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      # Keep running so we can see if other tests pass
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.6'
```

Comment:
A side note: python 3.6 is still listed in the docs, but this is something we may want to re-evaluate. See https://github.com/ome/omero-demo-cleanup/pull/6#issuecomment-1068307813

Reply:
Ah, I see it got bumped later. :+1: